docs/peekobot.js

```diff
@@ -1,120 +1,185 @@
-                    button.dataset.next = option.next;
-                }
-                choices.appendChild(button);
-            });
-            insertNewChatItem(choices);
-        } else if (step.next) {
-            printResponse(chat[step.next]);
-        }
-    };
-
-    const printChoice = function (choice) {
-        const choiceElem = document.createElement('div');
-        choiceElem.classList.add('chat-ask');
-        choiceElem.innerHTML = choice.innerHTML;
-        insertNewChatItem(choiceElem);
-    };
-
-    const disableAllChoices = function () {
-        const choices = document.querySelectorAll('.choice');
-        choices.forEach(function (choice) {
-            choice.disabled = 'disabled';
-        });
-        return;
-    };
-
-    const handleChoice = async function (e) {
-
-        if (!e.target.classList.contains('choice') || 'A' === e.target.tagName) {
-            // Target isn't a button, but could be a child of a button.
-            var button = e.target.closest('#peekobot-container .choice');
-
-            if (button !== null) {
-                button.click();
+class PeekoBot {
+  constructor(chat, options) {
+    this.chat = chat;
+
+    this.supportedLanguages = options?.supportedLanguages || Object.keys(this.chat);
+    this.currentLanguage = options?.currentLanguage || Object.keys(this.chat)[0];
+
+    this.peekobot = document.getElementById('peekobot');
+    this.container = document.getElementById('peekobot-container');
+    this.inner = document.getElementById('peekobot-inner');
+    this.restartButton = null;
+  }
+
+
+
+  sleep(ms) {
+      return new Promise(resolve => setTimeout(resolve, ms));
+  }
+
+  scrollContainer() {
+      this.inner.scrollTop = this.inner.scrollHeight;
+  }
+
+  insertNewChatItem(elem) {
+      //container.insertBefore(elem, peekobot);
+      this.peekobot.appendChild(elem);
+      this.scrollContainer();
+      //debugger;
+      elem.classList.add('activated');
+  }
+
+  async printResponse(step) {
+      const response = document.createElement('div');
+      response.classList.add('chat-response');
+      console.log(step);
+      response.innerHTML = step.text;
+      this.insertNewChatItem(response);
+
+      await this.sleep(1500);
+
+      if (step.options) {
+          const choices = document.createElement('div');
+          choices.classList.add('choices');
+          step.options.forEach(function (option) {
+              const button = document.createElement(option.url ? 'a' : 'button');
+              button.classList.add('choice');
+              button.innerHTML = option.text;
+              if (option.url) {
+                  button.href = option.url;
+              } else {
+                  button.dataset.next = option.next;
+              }
+              choices.appendChild(button);
+          });
+          this.insertNewChatItem(choices);
+      } else if (step.next) {
+          this.printResponse(chat[this.currentLanguage][step.next]);
```

Comment:
As you see, now we need to specify the current language in chat. Please replace `chat` with `this.chat`